plugins/modules/nd_setup.py

```diff
@@ -276,17 +276,17 @@
       The deployment mode cannot be changed after the cluster is deployed.
       Therefore, you must ensure that you have completed all service-specific prerequisites described in the early chapters of this guide
       U(https://www.cisco.com/c/en/us/td/docs/dcn/nd/3x/deployment/cisco-nexus-dashboard-and-services-deployment-guide-311.html)
-    - This option is only applicable for ND version 3.1.1 to 3.2.2.
+    - This option is only applicable for ND versions from 3.1.1 to 3.2.2.
     type: list
     elements: str
     choices: [ ndo, ndfc, ndi-virtual, ndi-physical ]
     aliases: [ mode ]
   external_services:
     description:
     - The persistent Service IPs/Pools to be provided.
-    - For ND versions between 3.1.1 and 3.2.2,
-      This can only be used when O(deployment_mode) includes V(ndi) or V(ndo) otherwise it will be ignored.
-    - This option is only applicable for ND version 3.1.1 and later.
+    - For ND versions between 3.1.1 and 3.2.2, This can only be used when O(deployment_mode)
```

Comment:
Can we change this to ->
- For ND versions between 3.1.1 and 3.2.2, this can only be used when O(deployment_mode)